<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exeInitSetting</class>
 <widget class="QWidget" name="exeInitSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>交互式练习设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/editIcon.png</normaloff>:/res/editIcon.png</iconset>
  </property>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正粗黑宋简体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认设置</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/ChooseIcon.png</normaloff>:/res/ChooseIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>方正粗黑宋简体</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>进行本次交互式练习设定：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="exeTime" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>等线</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>练习轮数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="exeTimeInput">
         <property name="font">
          <font>
           <family>等线</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="recordUsedTime">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>记录练习用时</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="exeScope">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>练习范围</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="A1">
         <property name="text">
          <string>平假名清音部分</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="A2">
         <property name="text">
          <string>平假名浊音部分</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="A3">
         <property name="text">
          <string>平假名拗音部分</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="B1">
         <property name="text">
          <string>片假名清音部分</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="B2">
         <property name="text">
          <string>片假名浊音部分</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="B3">
         <property name="text">
          <string>片假名拗音部分</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="FGA">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>每轮每组练习一定时间强制提交答案</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="FGAwaitTime" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>强制提交答案等待时间（秒）：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FGAwaitTimeInput">
           <property name="font">
            <font>
             <family>等线</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
